fixtures/src/components/controlled/checkbox.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Checkbox } from 'carbon-react/lib/__experimental__/components/checkbox';
+import { LogConsumer } from '../log';
+
+const ControlledCheckbox = () => {
+  const [checked, setState] = useState(false);
+  const [checkedExampleTwo, setStateExampleTwo] = useState(true);
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = store => (e) => {
+          store(e.target.checked);
+          log(e, { method: 'onChange' });
+        };
+        const onBlur = e => log(e, { method: 'onBlur' });
+        return (
+          <React.Fragment>
+            <h1>Controlled Checkbox</h1>
+            <h2>Unchecked</h2>
+            <div id='controlled_checkbox'>
+              <ul>
+                <li>onChange handler should update the log when each checkbox is checked</li>
+                <li>onBlur handler should update the log when each checkbox is blurred</li>
+                <li>each checkbox button has props value, name, id which should be reflected in both events</li>
```

Comment:
No `value` prop is passed to the `<Checkbox>`:
```suggestion
                <li>each checkbox button has props name, id which should be reflected in both events</li>
```